### PR TITLE
Public dns zones

### PR DESCRIPTION
**Pull request by uolter**
### List of changes

- production config is still to be changed. I used it to creta the public DNS zone.

### Type of changes

- [x] Add new resources
- [x] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [x] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

### Other information

---

### CODE REVIEW

File: src/env/prod/terraform.tfvars [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+db_sku_name       = "GP_Gen5_2"
+db_enable_replica = true
+dns_zone_prefix   = "cstar"
+ehns_sku_name     = "Standard"
+enable_azdoa      = true
+env_short         = "p"
+eventhubs = [
+  {
+    name              = "bpd-citizen-trx"
+    partitions        = 1
```
Comment (ElfoLiNk) [inline src/env/prod/terraform.tfvars:27]: Use correct values for PROD

File: src/env/prod/terraform.tfvars [hunk 2]
```diff
@@ -0,0 +1,171 @@
+apim_notification_sender_email = "info@pagopa.it"
+apim_publisher_email           = "io-operations@pagopa.it"
+apim_publisher_name            = "PagoPa Centro Stella DEV"
+apim_sku                       = "Developer_1"
+app_gateway_host_name          = "api.cstar-dev.pagopa.it"
```
Comment (ElfoLiNk) [inline src/env/prod/terraform.tfvars:5]: `app_gateway_host_name` probably is no more needed. Remove also the var from variables.tf

File: src/env/prod/terraform.tfvars (hunk 1 above)
Comment (uolter) [reply src/env/prod/terraform.tfvars:26]: I've just created the DNS public zone: apply with target. I've not created all resources yet.

File: src/env/prod/terraform.tfvars (hunk 2 above)
Comment (uolter) [reply src/env/prod/terraform.tfvars]: Done. Thank you.

**Pull request merged**